examples/dynamics/lennard_jones_potential_equilibrium_separation.py

```diff
@@ -0,0 +1,56 @@
+from sympy import solve, Eq, symbols
+from symplyphysics import (
+    print_expression,
+    vector_magnitude,
+    convert_to,
+    units,
+    Quantity,
+)
+from symplyphysics.core.fields.scalar_field import ScalarField
+from symplyphysics.core.points import cartesian_point
+from symplyphysics.laws.dynamics.fields import (
+    conservative_force_is_gradient_of_potential_energy as potential_law,
+)
+
+# Description
+## The potential energy of attraction between atoms and molecules can be modeled in the form of the
+## Lennard-Jones potential: U = A/r^12 - B/r^6, where A and B are positive constants. For argon,
+## the corresponding coefficients are A = 1.58e-134 J*m^12, and B = 1.02e-77 J*m^6. Find the equilibrium
+## separation, i.e. the distance between the atoms at which the force on each atom is zero.
+
+# Reference frame:
+## We assume two atoms lying on the x axis, and the first atom is located at the origin.
+## Thus, the position of the second atom is described by its position on the x axis.
+
+x = symbols("x", real=True)
+A, B = symbols("A B", positive=True)
+
+values = {
+    A: Quantity(1.58e-134 * units.joule * units.meter**12),
+    B: Quantity(1.02e-77 * units.joule * units.meter**6),
+}
+
+
+def potential_energy_function(point):
```

Comment:
Might be a good idea to move it to laws.

Reply:
I agree, maybe into a new branch, like `physical_chemistry`? It's more about molecular physics than simply dynamics

Reply:
We have a `chemistry` branch.

Reply:
Opened the corresponding [PR](https://github.com/blackyblack/symplyphysics/pull/289)